README.MD

```diff
@@ -1 +0,0 @@
-# GPostCard
```

Comment:
Make sure to not delete your previous Pull Request (README.MD).

Reply:
Everything is fine about that.